src/package.xml

```diff
@@ -1,112 +1,64 @@
-                <column name="y" displayName="Value for y" dataType="Double" validationType="Decimal" validationCondition="None" />
-                <column name="OutputRasterFile" displayName="Output Raster File" dataType="String" isExternalFile="True" externalFileFilter="GeoTIFF Files (*.tif)|*.tif|All Files (*.*)|*.*" isRaster="True" />
-              </columns>
-              <defaultRecords />
-            </datasheet>
-          </datasheets>
-        </datafeed>
-        <datafeed name="OutputDatasheet" displayName="Outputs" dataScope="Scenario">
-          <datasheets>
-            <datasheet name="OutputDatasheet" displayName="Outputs">
-              <columns>
-                <column name="OutputDatasheetID" dataType="Integer" isPrimary="True" />
-                <column name="ScenarioID" dataType="Integer" isCompositeIndex="True" />
-                <column name="Iteration" dataType="Integer" validationType="WholeNumber" validationCondition="GreaterEqual" formula1="1" format="d" isCompositeIndex="True" />
-                <column name="Timestep" dataType="Integer" validationType="WholeNumber" validationCondition="GreaterEqual" formula1="0" format="d" isCompositeIndex="True" />
-                <column name="yCum" displayName="Cumulative value for y" dataType="Double" validationType="Decimal" validationCondition="None" />
-              </columns>
-              <defaultRecords />
-            </datasheet>
-          </datasheets>
-        </datafeed>
-      </datafeeds>
-    </transformer>
-  </transformers>
-  <layouts>
-        <layout name="coreforms_ResultTransformers">
-            <item name="corestime_ChartTransformer"/>
-            <item name="corestime_MapTransformer"/>
-        </layout>
-        <layout name="coreforms_LibraryDatafeeds">
-            <group name="Options">
-                <item name="core_Rconfig"/>
-                <item name="core_Options" displayName="General"/>
-            </group>
-        </layout>
-        <layout name="coreforms_ScenarioDatafeeds">
-            <item name="RunControl"/>
-            <item name="InputDatasheet"/>
-            <item name="IntermediateDatasheet"/>
-            <item name="OutputDatasheet"/>
-        </layout>
-        <layout name="corestimeforms_Charts" configurationSheet="RunControl">
-            <item name="y" displayName="y" dataSheet="IntermediateDatasheet" column="y"/>
-            <item name="yCum" displayName="Cumulative y" dataSheet="OutputDatasheet" column="yCum"/>
-        </layout>
-        <layout name="corestimeforms_Maps" configurationSheet="RunControl">
-            <item name="RasterMap" displayName="Output Raster Map" dataSheet="IntermediateDatasheet" column="OutputRasterFile"/>
-        </layout>
-  </layouts>
-  <views />
+<package name="helloworldConda" displayName="Hello World Spatial (R)" description="Example demonstrating how to use spatial data with an R model" version="2.0.0" url="https://apexrms.github.io/helloworldEnhanced/">
```

Comment:
wrong displayName and description

Reply:
Also, the helloworldConda package is now being built off basic helloworld (not helloworldSpatial)